pkg/lib/facade/coordinator.go

```diff
@@ -1425,3 +1426,16 @@ func (c *Coordinator) GetUsersByStandardAttribute(attributeName string, attribut
 
 	return uniqueUserIDs, nil
 }
+
+func (c *Coordinator) GetUsersByLoginID(loginIDKey string, loginIDValue string) (string, error) {
+	loginID, err := c.Identities.GetByKeyAndValue(loginIDKey, loginIDValue)
+
+	if errors.Is(err, api.ErrIdentityNotFound) {
+		// User identity not found is not an error in this function.
+		return "", nil
```

Comment:
You need to return `api.UserNotFound` here.

Reply:
updated to return `api.ErrUserNotFound`